src/edu/csus/ecs/pc2/core/execute/IExecutableMonitor.java

```diff
@@ -0,0 +1,56 @@
+// Copyright (C) 1989-2019 PC2 Development Team: John Clevenger, Douglas Lane, Samir Ashoo, and Troy Boudreau.
+/**
+ * IExecutableMonitor interface
+ * This interface should be implemented by classes that use the Executable class.
```

Comment:
I don't know that the above is really an accurate statement; I don't think there's any imperative that a class which "uses the Executable class" necessarily NEEDS to use this interface; the purpose of the interface is to provide classes which use Executable with a mechanism to "monitor" the ongoing status of Executable operations -- but they don't HAVE to do that; they could simply invoke Executable methods and wait for them to complete without caring about monitoring ongoing status changes.  I think the comment is misleading, and/or incomplete in its description of the intention of the interface.

Reply:
Fixed and pushed.